### DISCUSSION TITLE
Model responds with "you didn't provide an image" or equivalent

### DISCUSSION THREAD

**Discussion by Rudd-O** [open-webui/open-webui#3283] [General]
# Bug Report

Various vision-capable models (Llama3, phi3) I've tried ignore the `images` parameter of the API call `/chat`.  They say things like:

```
As mentioned earlier, without an actual visual provided for "[img-0]" or "this one," it's impossible for me to give a description. However, if these are placeholders for images that should have been included with your message, please make sure the images are attached properly when you resubmit your query. Once they are visible, I can certainly assist in interpreting and describing them!
```

It was working first, and now it's no longer working.

No relevant log messages from ollama or Open-WebUI.

## Description

**Bug Summary:**

Chat messages with uploaded images are not recognized as having images.

**Steps to Reproduce:**

Select image (phone / desktop doesn't matter), type question about image, get error above.

**Expected Behavior:**

Get image description as I used to get.

**Actual Behavior:**

See above.

## Environment

- **Open WebUI Version:** ghcr.io/open-webui/open-webui:main at the time of this bug report
- **Ollama (if applicable):** 0.1.44 (installed today)

- **Operating System:** Lunix (Fedora 40)
- **Browser (if applicable):** Chromium on mobile, Firefox on desktop.

## Reproduction Details

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.

## Logs and Screenshots

**Browser Console Logs:**
[Include relevant browser console logs, if applicable]

**Docker Container Logs:**
[Include relevant Docker container logs, if applicable]

**Screenshots (if applicable):**
[Attach any relevant screenshots to help illustrate the issue]

## Installation Method

Docker.

## Additional Information

None.

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by tjbck** [comment]
![shrek-shrek-rizz](https://github.com/open-webui/open-webui/assets/25473318/50f9dc49-a287-4b70-9a54-52175706fa1e)

Love is blind 🧑‍🦯

**Comment by Rudd-O** [comment]
![Screenshot_20240619_021103](https://github.com/open-webui/open-webui/assets/346306/2a2df697-631c-4f88-993d-f4480cdc900e)

Self explanatory.  Models appear blind.

**Reply by justinh-rahb** [reply]
That's because you're not using a multimodal model. llama3 does not understand images. Neither does Phi3. There are _variants_ of both that do.. but you're not using them.

**Comment by Rudd-O** [comment]
OK. Tried with LLaVa and that worked fine.  I'm closing the comment.  The Vision checkbox should not be selected by default when adding a model.
